src/windows/config.js

```diff
@@ -297,7 +302,12 @@ const BROWSER_PLATFORMS = selectFrom([
     platform: 'p2-aarch64',
     platformFilter: {
       eq: {
-        platform: ['android-hw-p2-8-0-aarch64', 'android-hw-p2-8-0-android-aarch64'],
+        platform: [
+          'android-hw-p2-8-0-aarch64-shippable',
+          'android-hw-p2-8-0-android-aarch64-shippable',
+          'android-hw-p2-8-0-aarch64',
+          'android-hw-p2-8-0-android-aarch64',
+        ],
```

Comment:
`{"prefix":{"platform":"android-hw-p2-8-0-aarch64"}}`  is a bit easier to read, but no need to change